cmd/containerd/server/server.go

```diff
@@ -202,6 +202,13 @@ func New(ctx context.Context, config *srvconfig.Config) (*Server, error) {
 		}
 
 		tcpServerOpts = append(tcpServerOpts, grpc.Creds(credentials.NewTLS(tlsConfig)))
+	} else if config.GRPC.TCPTLSCName != "" {
```

Comment:
@dmcgowan @samuelkarp any thoughts on doing this? 
Windows uses certstore to load certificates which is a rather important Windows usecase which customers have been requesting for.